addons/mail/security/ir.model.access.csv

```diff
@@ -27,6 +27,10 @@ access_discuss_channel_rtc_session_system,discuss.channel.rtc.session.system,mod
 access_mail_alias_all,mail.alias.all,model_mail_alias,,0,0,0,0
 access_mail_alias_user,mail.alias.user,model_mail_alias,base.group_user,1,0,0,0
 access_mail_alias_system,mail.alias.system,model_mail_alias,base.group_system,1,1,1,1
+access_mail_alias_domain_public,mail.alias.domain.public,model_mail_alias_domain,base.group_public,1,0,0,0
+access_mail_alias_domain_portal,mail.alias.domain.portal,model_mail_alias_domain,base.group_portal,1,0,0,0

```

Comment:
Do public/portal really need to be able to read those?

Also, when the rights are equal we usually don't repeat them, but I know you like to be specific sometimes so that may be why.

EDIT: looks like public lost the right in the following commit, still a bit interested to know why portal needs to have it